setup.py

```diff
@@ -134,6 +155,7 @@ def rglob(path, glob):
         "Operating System :: Unix",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
```

Comment:
```suggestion
        "Programming Language :: Python :: 3.9",
        "Programming Language :: Python :: 3.10",
```

Reply:
(I realized I forgot to add it in #726)

Reply:
Sorry, I don't know why this is showing up here. Basically just make sure that the supported Python versions are 3.7, 3.8, 3.9, 3.10.